api/envoy/config/trace/v3/opentelemetry.proto

```diff
@@ -17,10 +21,41 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 // Configuration for the OpenTelemetry tracer.
 //  [#extension: envoy.tracers.opentelemetry]
 message OpenTelemetryConfig {
-  // The upstream gRPC cluster that will receive OTLP traces.
-  // Note that the tracer drops traces if the server does not read data fast enough.
-  // This field can be left empty to disable reporting traces to the collector.
-  core.v3.GrpcService grpc_service = 1;
+  // Configuration for sending traces to the collector over HTTP. Note that
+  // this will only be used if the grpc_service is not set above.
+  message HttpConfig {
+
+    // The upstream cluster that will receive OTLP traces over HTTP.
+    string cluster_name = 1;
+
+    // The path to the trace ingest endpoint. The path is appended with the host name configured in the cluster.
+    // Optional: If omitted, "/v1/traces" will be used as default.
+    string traces_path = 2;
+
+    // Sets the maximum duration in milliseconds that a response can take to arrive upon request.
+    google.protobuf.Duration timeout = 3 [(validate.rules).duration = {
+      required: true
+      gte {}
+    }];
+
+    // Custom header values to be added to the OTLP HTTP request.
+    repeated config.core.v3.HeaderValue headers = 4;
```

Comment:
I found in other components in the code the usage of this type to specify custom headers. I think the end result turned out quite nice and very similar to how one configures custom headers in the HTTP exporter in the OpenTelemetry collector. An example might be this:

```
tracing:
  provider:
    name: envoy.tracers.opentelemetry
    typed_config:
      "@type": type.googleapis.com/envoy.config.trace.v3.OpenTelemetryConfig
      service_name: envoy-HTTP-exporter
      http_config:
        cluster_name: my_backend
        hostname: "my-backend.com"
        headers:
          - key: "Authorization"
            value: "auth-token"
          - key: "other-header"
            value: "custom value"
        timeout: 5s
```